docs/opcodes/F5.mdx

```diff
@@ -5,17 +5,22 @@ group: System operations
 
 ## Notes
 
-This instruction has no effect on the EVM state. See [here](https://ethereum.org/en/developers/docs/smart-contracts/anatomy/#events-and-logs).
+Equivalent to [CREATE](/#F0), except the salt allows to be created at a deterministic address.
```

Comment:
I realise the sentence is not complete: "the salt allows *the new contract* to be created

Reply:
+1 I changed it